### PR TITLE
Normalize behavior of "getText" method

### PR DESCRIPTION
**Pull request by aik099**
See https://github.com/minkphp/driver-testsuite/issues/98.

### CODE REVIEW

File: src/WebdriverClassicDriver.php
Comment (uuf6429) [inline src/WebdriverClassicDriver.php:300]: Is the `' '` a non-breaking space? I think I would rather use an escaped sequence (`"\xA0"`), otherwise it might not be obvious.

Comment (oleg-andreyev) [reply src/WebdriverClassicDriver.php:300]: @uuf6429 was writting the same, but if it's NBSP (non-breaking space) it should be U+202F

so it could
```php
mb_chr(8239)
``` 

or 

```php
chr(0xE2) . chr(0x80) . chr(0xAF)
```

no?

Comment (aik099) [reply src/WebdriverClassicDriver.php]: Done. However, I had to use `"\xc2\xa0"`, because both `"\xA0"` and `chr(0xA0)` produced a broken UTF-8 symbol: `�`.

The https://stackoverflow.com/questions/40724543/how-to-replace-decoded-non-breakable-space-nbsp helped.

Comment (aik099) [reply src/WebdriverClassicDriver.php]: Both editors I've had (PhpStorm and Sublime Text) were showing `0xa0` and not `0xc20xa0`:
PhpStorm_NBSP
SublimeText_NBSP

In PHP you can't get it working without adding `\xc2` first. I wonder if it works in Selenium code at all without `\xC2`.

Comment (uuf6429) [reply src/WebdriverClassicDriver.php]: I think I'm fine with it as long as the tests pass...should we maybe wait until the driver-testsuite one is merged and then rerun this?

**Review by uuf6429 — APPROVED** [review]

File: src/WebdriverClassicDriver.php
Comment (aik099) [reply src/WebdriverClassicDriver.php]: > I think I'm fine with it as long as the tests pass...should we maybe wait until the driver-testsuite one is merged and then rerun this?

Done.

See https://github.com/minkphp/webdriver-classic-driver/actions/runs/12132940669/job/33827642410?pr=52:

GitHubActions_GetTextTest

**Comment by aik099** [comment]
@uuf6429 , please do a bugfix release in the foreseeable future (this month) if possible.

**Pull request merged**
